pkg/scheduler/schedulercache/cache.go

```diff
@@ -112,6 +114,15 @@ func (cache *schedulerCache) UpdateNodeNameToInfoMap(nodeNameToInfo map[string]*
 	cache.mu.Lock()
 	defer cache.mu.Unlock()
 	for name, info := range cache.nodes {
+		if utilfeature.DefaultFeatureGate.Enabled(features.BalanceAttachedNodeVolumes) {
+			// Transient scheduler info is reset here.
+			if info.TransientInfo != nil {
+				info.TransientInfo.resetTransientSchedulerInfo()
+			} else if info.TransientInfo == nil {
```

Comment:
You do not need the "if" part.

In fact, given that `TransientInfo` is initialized with nodeInfo, please remove the whole `else` part and not initialize it here.

Reply:
Done Bobby. PTAL.